### DISCUSSION TITLE
How to set code coverage per component/directory/path

### DISCUSSION THREAD

**Discussion by mfaridzia** [vitest-dev/vitest#4256] [Q&A] [answered]
In jest i can set code coverage like branches, statement etc per component/directory/path:
```
const config = {
  coverageThreshold: {
    global: {
      branches: 50,
      functions: 50,
      lines: 50,
      statements: 50,
    },
    './src/components/finance': {
      branches: 40,
      statements: 40,
    },
  },
};
```

Is that possible in Vitest? Thanks

**Comment by AriPerkkio** [comment, marked as answer]
This is not yet implemented. There is a feature request for this at https://github.com/vitest-dev/vitest/issues/3709.

**Reply by mfaridzia** [reply]
I see, thank you @AriPerkkio
